open_academy/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+open_academy.access_openacademy_course,access_openacademy_course,open_academy.model_openacademy_course,base.group_user,1,0,0,0
+open_academy.access_openacademy_session,access_openacademy_session,open_academy.model_openacademy_session,base.group_user,1,0,0,0
```

Comment:
This need all the permissions because the user can't create or write these records.